src/views/SslList.vue

```diff
@@ -274,8 +274,11 @@ export default defineComponent({
         {
           title: 'AWS',
           fieldNames: ['links'],
-          displayFunction: (item) => {
-            return item?.links?.provider === 'aws' ? 'true' : 'false'
+          displayFunction: (item: Certificate) => {
+            const isAWS = _.some(item?.links, (link: Link) => {
+              return link?.provider === 'aws' ? true : false
```

Comment:
return statement can be simplified

```suggestion
              return link?.provider === 'aws'
```

Reply:
Done